user_guide/content/modules/about_areas.adoc

```diff
@@ -1,4 +1,8 @@
 [#about_areas]
 = About Areas
 
-*Areas* are used to organize related <<about_work_items,Work Items>> into groups, to distinguish between different types or functionalities that are being worked on within a <<about_collaboration_spaces,Collaboration Space>>.
+*Areas* are used to organize related <<about_work_items,Work Items>> into groups to distinguish between different types or functionalities that are being worked on within a <<about_collaboration_spaces,Collaboration Space>>.
+
+A Space can also include multiple Areas. As an example, your Collaboration Space represents your software project, and each Area tracks various components attached to the Space.
+
+Each Area can have multiple child Areas, which represents larger components for your software project including multiple smaller components.
```

Comment:
`s/represents/represent/`
`s/including/, including/`

Reply:
Suggest: which represents larger components for your software project including multiple smaller components.>representing sub-components of the larger components. 
Since you have already mentioned 'area' mapping to larger component in the previous sentence.

Reply:
Incorporated both bits of feedback in the updated commit, I think. I didn't include s/represents/represent since the plural form should be correct here (subject is singular [Collab Space])